pkg/netns/netns_linux.go

```diff
@@ -28,15 +28,25 @@ import (
 	"sync"
 
 	"github.com/containernetworking/plugins/pkg/ns"
+	"github.com/containers/libpod/pkg/rootless"
 	"golang.org/x/sys/unix"
 )
 
-const nsRunDir = "/var/run/netns"
+// get NSRunDir returns the dir of where to create the netNS. When running
+// rootless, it needs to be at a location writable by user.
+func getNSRunDir() string {
+	if rootless.IsRootless() {
+		return "/tmp/netns"
```

Comment:
it should be `filepath.Join(util.GetRootlessRuntimeDir(), "netns")`

Reply:
Thank you! Fixing